docs/howto/django.rst

```diff
@@ -5,36 +5,99 @@ Install procrastinate with:
 
 .. code-block:: console
 
-    $ (venv) pip install 'procrastinate[django]'
+    $ (venv) pip install procrastinate 'procrastinate[django]'
```

Comment:
Hm ? I'm pretty very absolutely sure that `pip install procrastinate 'procrastinate[django]'` installs procrastinate alongside with the extras marked as `django` in the packaging, so adding `procrastinate` doesn't change anything, and makes the line slightly more complicated. Is there something I missed ?

(also, the simple quotes are important on that line. If you tried without the simple quotes, maybe that's why you didn't get the expected result ?)

Reply:
Probably my bad - I had problems installing extras with pip (or it was my fault, or a bug that were fixed already). I'll change back to the first version.